### PR TITLE
style(console): fix multi-text input margin

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Fix the `MultiTextInput` according to the design file:
image
https://www.figma.com/file/hqAWH3Di8gkiV5TXAt6juO/%F0%9F%8C%B9-%5BAC%5D-Layout-UI-Optimization?node-id=174%3A23222&t=UFwmBDbVBML13YqB-4

### Before
image

### After
multi-text-input-in-guide
image
image

### CODE REVIEW

File: packages/console/src/components/MultiTextInput/index.module.scss [hunk 1]
```diff
@@ -5,14 +5,22 @@
     margin-top: _.unit(2);
   }
 
+  > :first-child {
+    &:not(:nth-last-child(2)) {
+      padding-right: _.unit(9);
+    }
+  }
```
Comment (charIeszhao) [inline packages/console/src/components/MultiTextInput/index.module.scss:12]: This is a bit weird, why?

File: packages/console/src/components/MultiTextInput/index.module.scss [hunk 2]
```diff
@@ -5,14 +5,20 @@
     margin-top: _.unit(2);
   }
 
+  .firstFiledWithMultiInputs {
```
Comment (charIeszhao) [inline packages/console/src/components/MultiTextInput/index.module.scss:8]: type: filed => field

File: packages/console/src/components/MultiTextInput/index.tsx [hunk 3]
```diff
@@ -47,10 +58,15 @@ const MultiTextInput = ({ title, value, onChange, onKeyPress, error, placeholder
...
 
   return (
-    <div className={styles.multilineInput}>
+    <div className={classNames(styles.multilineInput, className)}>
       {fields.map((fieldValue, fieldIndex) => (
-        // eslint-disable-next-line react/no-array-index-key
-        <div key={fieldIndex}>
+        <div
+          // eslint-disable-next-line react/no-array-index-key
+          key={fieldIndex}
```
Comment (charIeszhao) [inline packages/console/src/components/MultiTextInput/index.tsx:65]: I don't think we should use indexes as keys, as the inputs might be deleted and the indexes would be changed

Comment (xiaoyijun) [reply packages/console/src/components/MultiTextInput/index.tsx:65]: This is a special case, for the value is `string[]`, and if any input is deleted, the whole component will be re-rendered.

File: packages/console/src/components/MultiTextInput/index.module.scss (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/components/MultiTextInput/index.module.scss]: updated.

File: packages/console/src/components/MultiTextInput/index.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/components/MultiTextInput/index.module.scss]: Updated.

File: packages/console/src/components/MultiTextInputField/index.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  formFieldClassName,
+  value,
+  ...rest
+}: Props) => (
+  <FormField
+    title={title}
+    isRequired={isRequired}
+    tooltip={tooltip}
+    className={formFieldClassName}
+    headlineClassName={conditional(value && value.length > 1 && styles.headlineWithMultiInputs)}
```
Comment (charIeszhao) [inline packages/console/src/components/MultiTextInputField/index.tsx:28]: nit: `value?.length`

**Review by charIeszhao — APPROVED** [review]

File: packages/console/src/components/MultiTextInputField/index.tsx (hunk 4 above)
Comment (xiaoyijun) [reply packages/console/src/components/MultiTextInputField/index.tsx:28]: We can't.
image

**Pull request merged**